### ISSUE TITLE
Extend channels support to whisper, guild, say, etc.

### ISSUE THREAD

**Issue by maksxpl** [Krappa322/arcdps_unofficial_extras_releases#25]
It would be quite cool if devs could easily play with all chat channels, instead of just party/squad. (as I'm mainly asking due to [chat-log](https://github.com/cheahjs/arcdps-chat-log) addon which could benefit from this for sure)

**Comment by cubimon** [comment]
Yes please, I would love to have something like that for the upcoming wvw alliance change to implement some kind of user validation based on map/say chat. Since the API probably won't work for a while.

**Comment by anywayaura** [comment]
would appreciate npc speech text  🙏🏻

**Comment by KirillSerogodsky** [comment]
> would appreciate npc speech text 🙏🏻

Really needed for the translator addon

**Comment by knoxfighter** [comment]
I sat down and reversed the chat further, so i added a callback for NPC Chatter/Speech.
It is available with the [latest pre-release](https://github.com/Krappa322/arcdps_unofficial_extras_releases/releases/tag/v1.25.rc1).
The new callback superseeds the old one and can replace it. Please test it and report back any issues and suggestions.
